tasks/util/container_commands.js

```diff
@@ -59,11 +59,13 @@ function getRunCI(commands) {
     commands = ['export CIRCLECI=1', containerCommands.cdHome].concat(commands);
 
     var commandsJoined = '"' + commands.join(' && ') + '"';
+    var containerId = '$(docker inspect --format \'{{.Id}}\' ' + constants.testContainerName + ')';
 
     return [
         'sudo',
-        'lxc-attach -n',
-        '$(docker inspect --format \'{{.Id}}\' ' + constants.testContainerName + ')',
+        'lxc-attach',
+        '-n', containerId,
+        '-f', '/var/lib/docker/containers/' + containerId + '/config.lxc',
```

Comment:
:speaker: Big shoutout to `hasimo` on CircleCI forums:

https://discuss.circleci.com/t/container-is-not-defined-on-lxc-attach/14684/5